api/controllers/StatsController.js

```diff
@@ -148,7 +148,7 @@ module.exports = {
       });
       const currentSeasonGames = Game.find({
         status: gameService.GameStatus.FINISHED,
-        updatedAt: { '>': currentSeason.startTime, '<': currentSeason.endTime },
+        updatedAt: { '>': dayjs(currentSeason.startTime).valueOf(), '<': dayjs(currentSeason.endTime).valueOf() },
```

Comment:
I wonder if It makes sense to change `updatedAt` and `createdAt` to the same timestamptz just to keep everything consistent. This works but I'm not sure it's the best way

Reply:
Yes I think we want all date formates to become `timestampz`